src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,441 @@
+For accessibility support, please [read the React beautiful dnd guidelines](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/about/accessibility.md).
+
+import TableDraggable from '../examples/tables/TableDraggable.tsx';
+import TableDraggableCode from '!!raw-loader!../examples/tables/TableDraggable.tsx';
+
+<CodeExample code={TableDraggableCode}>
+  <TableDraggable />
+</CodeExample>
+
+#### Table caption
+
+The [Caption](/components/table#table-caption) is the title of a table.
+
+A table must have a [Caption](/components/table#caption) or aria-describedby attribute that identifies the element which
+titles or describes the table
+
+import TableCaption from '../examples/tables/TableCaption.tsx';
+import TableCaptionCode from '!!raw-loader!../examples/tables/TableCaption.tsx';
+
+<CodeExample code={TableCaptionCode}>
+  <TableCaption />
+</CodeExample>
+
+#### Anchors
+
+Use an [Anchor](/components/anchor) for data cell text elements that link elsewhere.
+
+![Anchors](figma:patterns-tables-basic-formatting-anchors.png)
+
+1. These anchors within a table typically navigate the user to a new page
+1. If you want to open a link in an external window or tab, use an [External Anchor](/components/anchor#external-anchor)
+
+#### Date and Time
+
+Follow our [date and time formatting](/content/grammar#date-and-time) guidelines.
+
+![Date and Time](figma:patterns-tables-basic-formatting-date-time.png)
+
+## States
+
+### Rows and cells
+
+![Row and Cell States](figma:patterns-tables-basic-formatting-row-cell-states.png)
+
+<table>
+  <thead>
+    <tr>
+      <th width="30%">Row/Cell state</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1. Empty Cell</td>
+      <td>Cells with no values should be left blank. Do not use a hyphen, en, or em dash.</td>
+    </tr>
+    <tr>
+      <td>2. Hover</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>3. Focused</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>4. Selected</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>5. Focused and Selected</td>
+      <td>
+        The last row checked is the focused row, unless the user clicks on a different row or uses
+        the keyboard to change focus.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Loading
+
+Skeleton screens are used to indicate that a screen is loading. They are perceived as being shorter in duration when
+compared against a blank screen or a spinner.
+
+- Implement the [Skeleton](/components/skeleton) component in each cell up to three rows
+- For a table with three or more rows, show the first three rows as skeletons with the header cells being shown at all
+  times to provide instant context of incoming tabular data
+- The width of the Skeleton should match the width of the intended cell content, while the Skeleton’s height will be 60%
+  of the font size
```

Comment:
Just want to double-check that we want to call out the "60%" unit. There have been previous concerns re: Garden's `Skeleton` implementation being too slim and not accurately simulating the rendered text. If we address that concern, we'll have to keep this text up-to-date.

Reply:
@jzempel I'm not familiar with that concern. Previous pattern documentation by Allison pointed to the 60% and it wasn't raised in any prior design reviews of this content. Happy to update this whether that be in this release or an iteration in the future based on the data at hand.